docs/src/svelte.md

```diff
@@ -2,137 +2,197 @@
 
 # @tomic/svelte
 
-An implementation of Atomic Data for [Svelte](https://svelte.dev/).
-This library is still at an early stage and the API is subject to change.
+An AtomicServer client for [Svelte](https://svelte.dev/).
+Makes fetching AtomicData easy.
+Fetched resources are chached and reactive, they will update when the data changes, even when the resource was changed by someone else.
 
```

Comment:
mention older svelte version without runes and max supported version of `@tomic/svelte`